src/components/CharactersPage/CharactersTable/CharactersTable.js

```diff
@@ -0,0 +1,88 @@
+import { useSelector } from "react-redux"
+import { isFinite } from "lodash"
+
+import CustomTable from "@/components/CustomTable"
+import { StyledTableCell, NoValueCell } from "../../CustomTable/Cells"
+import * as characterSelectors from "@/selectors/characterSelectors"
+import replaceIfNull from "@/utils/replaceIfNullUtils"
+
+const CharactersTable = ({ onRowClick }) => {
+  const characters = useSelector(characterSelectors.characters)
+
+  const columns = [
+    {
+      header: "Name",
+      field: "name",
+      cellRenderer: ({ cellData }) => {
+        if (!cellData) return <NoValueCell />
+        return <StyledTableCell value={cellData} />
+      },
+    },
+    {
+      header: "Eye Color",
+      field: "eyeColor",
+      cellRenderer: ({ cellData }) => {
+        if (!cellData) return <NoValueCell />
+        return <StyledTableCell value={cellData} />
+      },
+    },
+    {
+      header: "Height",
+      field: "height",
+      cellRenderer: ({ cellData }) => {
+        if (!isFinite(cellData)) return <NoValueCell />
+        return <StyledTableCell value={cellData} />
+      },
+    },
+    {
+      header: "Gender",
+      field: "gender",
+      cellRenderer: ({ cellData }) => {
+        if (!cellData) return <NoValueCell />
+        return <StyledTableCell value={replaceIfNull(cellData)} />
+      },
+    },
+    {
+      header: "Birth Year",
+      field: "birthYear",
+      cellRenderer: ({ cellData }) => {
+        if (!cellData) return <NoValueCell />
+        return <StyledTableCell value={replaceIfNull(cellData)} />
```

Comment:
You are repeating these `!cellData` checks too many times, move this logic into `StyledTableCell`.